backend/src/main/java/capstoneexpensetracker/backend/travelergroup/TravelerGroupService.java

```diff
@@ -29,4 +30,15 @@ public TravelerGroup addTravelerGroup(NewTravelerGroup newTravelerGroup) {
 
         return travelerGroupRepository.save(travelerGroup);
     }
+
+    public TravelerGroup deleteTravelerGroup(String id) {
+        TravelerGroup travelerGroupToFind = travelerGroupRepository
```

Comment:
idk i think you can delete this find all stream, because the method deleteById from Mongo give the same so you can saving here 5 rows code.